test/db/cmd/cmd_list

```diff
@@ -1,7 +1,7 @@
 NAME=List Parser Plugins Lp
 FILE=-
 CMDS=<<EOF
-Lp
+Lp | sort
```

Comment:
Dont use system commands. You can sort with the ~ command too

Reply:
Resolved